tests/serialization/convert-kicad-json-into-sexpression.test.ts

```diff
@@ -1,23 +1,22 @@
-import test from "ava"
-import { parseKicadModToKicadJson } from "src"
-import { convertKicadJsonToKicadMod } from "src/convert-kicad-json-to-kicad-mod"
-import { serializeSExpression } from "src/utils/serialize-s-expression"
-import { getTestFixture } from "tests/fixtures/get-test-fixture"
+import test from "ava";
+import { parseKicadModToKicadJson } from "src";
+import { convertKicadJsonToKicadMod } from "src/convert-kicad-json-to-kicad-mod";
+import { getTestFixture } from "tests/fixtures/get-test-fixture";
 
 test("convert kicad json to sexpression", async (t) => {
-  const fixture = await getTestFixture(t)
+    const fixture = await getTestFixture(t);
 
-  const fileContent = fixture.getKicadFile("Adafruit_Feather.kicad_mod")
+    const fileContent = fixture.getKicadFile("Adafruit_Feather.kicad_mod");
 
-  console.log("KICAD MOD INPUT", fileContent)
+    // console.log("KICAD INPUT:", fileContent);
 
-  const kicadJson = parseKicadModToKicadJson(fileContent)
+    const kicadJson = parseKicadModToKicadJson(fileContent);
 
-  console.log("KICAD JSON:", kicadJson)
+    // console.log("KICAD JSON:", kicadJson);
 
-  const kicadMod = convertKicadJsonToKicadMod(kicadJson)
+    const kicadMod = convertKicadJsonToKicadMod(kicadJson);
 
-  console.log("KICAD MOD OUTPUT", kicadMod)
+    console.log("KICAD MOD OUTPUT", kicadMod);
 
-  t.pass()
-})
+    t.pass();
+});
```

Comment:
Good job, you're using a slightly different formatter though and its messing with the diffs. We use "semi": false for all tecircuit projects (we also use the biome formatter rather than prettier)